conversions/weight_conversion.py

```diff
@@ -0,0 +1,64 @@
+"""
+Functions useful for conversion fo weight units
+"""
+
+
+def weight_conversion(from_type: str, to_type: str, value: float) -> any:
+    """
+    Conversion of weight unit with the help of kilogram_chart
+
+    "kilogram" : 1,
+    "gram" : pow(10, 3),
+    "milligram" : pow(10, 6),
+    "metric-ton" : pow(10, -3),
+    "long-ton" : 0.0009842073,
+    "short-ton" : 0.0011023122,
+    "pound" : 2.2046244202,
+    "ounce" : 35.273990723,
+    "carrat" : 5000,
+    "atomic-mass-unit" : 6.022136652E+26
+
+    Wikipedia reference: https://en.wikipedia.org/wiki/Conversion_of_units
```

Comment:
Please include this link in the module-level docstring as well.

Reply:
I don't get you can u explain (module-level docstring)?

Reply:
Module level docstring is
```python
"""
This is a module level docstring
"""
class someclass():
    """
    This is class level docstring
    """
```